src/gui/QvisHelpWindow.C

```diff
@@ -1417,7 +1421,27 @@ QvisHelpWindow::displayPage(const QString &page, bool reload)
         QString file(CompleteFileName(page));
         if(QFile(file).exists())
         {
-            helpBrowser->setSource(QUrl::fromLocalFile(file));
+            //
+            // We open the VisIt manuals using the user's default browser. All
+            // other pages are rendered inside of the widget using Qt.
+            //
+            if (page == manualPath)
+            {
+                helpBrowser->clear();
+                if (!QDesktopServices::openUrl(QUrl::fromLocalFile(file)))
+                {
+                    //
+                    // TODO: where do these messages get displayed??
+                    //
+                    Message(tr("The VisIt Manuals were unable to be "
```

Comment:
Message should be written to the Message window in the gui (there's an icon in the lower-right corner of the main window).  It may not pop-up if the message isn't an error, but IIRC the icon changes color when new info is posted. If you weren't seeing the message, perhaps this should be an Error instead of Message?

Reply:
Ah! I see it now. Thanks for explaining that. I was originally thinking an error message might be better, but I can't seem to find examples of how errors are thrown in the GUIs. Any tips or ideas of where to look?

Reply:
It looks like FileServerList.C throws exceptions. I'm not sure if throwing an exception is appropriate here, though.

Reply:
Just change 'Message' to 'Error'.

Reply:
Lol wow. Thanks for pointing this out!